$OpenBSD: patch-openbox_client_c,v 1.3 2017/09/25 21:34:28 dcoppa Exp $

Fix a NULL pointer dereference that was causing a crash when launching
Java GUI applications.
From https://bugzilla.icculus.org/show_bug.cgi?id=5277 via Arch
Linux.

Index: openbox/client.c
--- openbox/client.c.orig
+++ openbox/client.c
@@ -941,7 +941,7 @@ static ObAppSettings *client_get_settings_state(ObClie
                  !g_pattern_match(app->role,
                                   strlen(self->role), self->role, NULL))
             match = FALSE;
-        else if (app->title &&
+        else if (app->title && self->title &&
                  !g_pattern_match(app->title,
                                   strlen(self->title), self->title, NULL))
             match = FALSE;
